guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -21,7 +21,7 @@ See {ManagingHostsDocURL}creating-a-job-template_managing-hosts[Creating a Job T
 . Optional: To configure advanced settings for the job, click *Display advanced fields*.
 Some of the advanced settings depend on the job template, the following settings are general:
 
-* *Effective user* defines the user for executing the job, by default it is the SSH user.
+* *Effective user* defines the user for executing the job, by default it is the Script user.
```

Comment:
I'm not sure how to handle this, but there are scenarios in which this is not true. When ssh is used as a transport this holds, but when the pull provider is used, then the default user is `root`